www/docs/en/dev/guide/platforms/electron/index.md

```diff
@@ -29,11 +29,13 @@ Electron is a framework that uses web technologies (HTML, CSS, and JS) to build
 
 ### Linux
 
-**@todo Add Linux Requirements here.**
+* **Python** version 2.7.x. it is recommended to check your Python version, since some distributions like CentOS 6.x still use Python 2.6.x, 
```

Comment:
grammar

Reply:
Do you mean the comma at the end?

Reply:
No, probably uppercasing the `It` should fix it.

Reply:
Ohh, okay, I'll fix it.